content_handler/vulkan_rasterizer.cc

```diff
@@ -109,7 +103,7 @@ sk_sp<SkSurface> VulkanRasterizer::VulkanSurfaceProducer::ProduceSurface(
   auto sk_surface = SkSurface::MakeFromBackendRenderTarget(
       context_.get(), desc, SkColorSpace::MakeSRGB(), &props);
   if (!sk_surface) {
-    FTL_DLOG(INFO) << "MakeFromBackendRenderTarget Failed";
+    FTL_LOG(ERROR) << "MakeFromBackendRenderTarget Failed";
```

Comment:
Did you mean to keep the conversion of the `FTL_DLOG` to `FTL_LOG` in cases where the method returns a `null` value or otherwise does some cleanup? `flutter/vulkan` had to be safe from assertions during Vulkan setup because we didn't want to assert out before trying the OpenGL and software rendering fallbacks.

I guess it doesn't matter in case of the content handler. So don't have a strong opinion about it here.